readthedocs/config/config.py

```diff
@@ -692,6 +727,15 @@ def validate_search(self):
 
         return search
 
+    def validate_incompatible_keys(self):
+        # `formats` and `build.jobs.build.*` can't be used together.
```

Comment:
I would remove this restriction. It seems pretty common to use the default PDF builder (via `formats: [pdf]`) , but override `build.jobs.build.html`. What is the problem this restriction solves?